plugins/easyimage/plugin.js

```diff
@@ -6,6 +6,102 @@
 ( function() {
 	'use strict';
 
+	function isSideImage( widget ) {
+		return widget.element.hasClass( 'easyimage--side' );
```

Comment:
We're not using double dash in CSS class names. How about just `side-image`?

Reply:
Maybe `.easyimage-side`? It'll contain plugin name then.

Reply:
Sounds good to me.